### DISCUSSION TITLE
Where do I make a request to contribute a new front end language.

### DISCUSSION THREAD

**Discussion by arunshekher** [home-assistant/frontend#13678] [Localization] [answered]
May I know where to make a request for new language contribution if the language is not listed in Lokalize.

**Comment by matthiasdebaat** [comment, marked as answer]
Awesome you want to manage a new language. You can find this info on our developers website https://developers.home-assistant.io/docs/translations#adding-a-new-language

**Reply by arunshekher** [reply]
But how do I add a new language in Localize. I see no option to do that, could you help me on this? This is the language I want to add.

Type: language
Subtag: ml
Description: Malayalam
Added: 2005-10-16
Suppress-Script: Mlym

**Reply by arunshekher** [reply]
Thank you and apologies for the whining tone of my previous post. Gone through the posts at-lease 10 times in the past 14 days mindlessly before I figured out the solution.
